build.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import subprocess
+import shutil
+import sys
+import tempfile
+
+from jinja2 import Environment, FileSystemLoader
+
+jinja_env = Environment(
+    loader=FileSystemLoader('templates')
+)
+
+
+def build_dockerfile(sha1, version, tags):
+    build_dir = tempfile.mktemp()
+    shutil.copytree("docker", build_dir)
+    template = jinja_env.get_template("Dockerfile.jinja2")
+    dockerfile_content = template.render(sha1=sha1, version=version)
+    with open(os.path.join(build_dir, "Dockerfile"), "w") as dockerfile:
+        dockerfile.write(dockerfile_content)
+    build_command = ["docker", "build", "."]
+    for tag in tags:
+        build_command.extend(["-t", f"factoriotools/factorio:{tag}"])
+    try:
+        subprocess.run(build_command, cwd=build_dir, check=True)
+    except subprocess.CalledProcessError:
+        print("Build of image failed")
```

Comment:
Does this swallow the error log?

Reply:
Nope, see https://github.com/StopMotionCuber/factorio-docker/runs/2043351161